lib/Settings/contact.dart

```diff
@@ -0,0 +1,66 @@
+import 'package:flutter/material.dart';
+import 'package:get/utils.dart';
+import 'package:intl/intl.dart';
+import 'package:http/http.dart' as http;
+
+class ContactScreen extends StatefulWidget {
+  const ContactScreen({Key? key}) : super(key: key);
+
+  @override
+  State<ContactScreen> createState() => _ContactScreenState();
+}
+
+class _ContactScreenState extends State<ContactScreen> {
+
+  TextEditingController controller = TextEditingController();
+  TextEditingController controller1 = TextEditingController();
+  TextEditingController controller2 = TextEditingController();
```

Comment:
Change the names of the controllers. ex: `emailController`

Reply:
@bhattabhi013 changed accordingly